tests/Webapi/Dom/Webapi__Dom__MutationObserver__test.res

```diff
@@ -0,0 +1,31 @@
+open Webapi.Dom
+open MutationObserver
+
+let el = document->Document.createElement("strong")
+let observer: t = MutationObserver.make((_records, _observer) => ())
+
+let _ = observer->MutationObserver.observe(el, {
```

Comment:
Should we make this more type safe while we are at it. I doubt this will change much. It could be it's own opaque type with a constructor function or something. Like `makeObserveOptions(~subtree=true, ())` ?

Reply:
I considered that but it would be a breaking change; I meant to log a new task for it. #61